File: forge/src/main/java/dev/aika/bigclever/forge/BigCleverForge.java
package dev.aika.bigclever.forge;

import dev.aika.bigclever.forge.config.ClothConfig;
import net.minecraftforge.api.distmarker.Dist;
import net.minecraftforge.fml.DistExecutor;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;

import dev.aika.bigclever.BigClever;

@Mod(BigClever.MOD_ID)
public final class BigCleverForge {
    public BigCleverForge() {
        FMLJavaModLoadingContext.get().getModEventBus().register(this);

        BigClever.init();

        DistExecutor.safeRunWhenOn(Dist.CLIENT, () -> ClothConfig::setupConfig);
    }
}

File: forge/src/main/kotlin/dev/aika/bigclever/forge/BigCleverForge.kt
package dev.aika.bigclever.forge

import dev.aika.bigclever.BigClever
import dev.aika.bigclever.forge.config.ClothConfig
import net.minecraftforge.api.distmarker.Dist
import net.minecraftforge.fml.DistExecutor
import net.minecraftforge.fml.DistExecutor.SafeRunnable
import net.minecraftforge.fml.common.Mod
import thedarkcolour.kotlinforforge.forge.MOD_BUS

@Suppress("unused")
@Mod(BigClever.MOD_ID)
class BigCleverForge {
    init {
        MOD_BUS.register(this)

        BigClever.init()

        DistExecutor.safeRunWhenOn(Dist.CLIENT) { SafeRunnable(ClothConfig::setupConfig) }
    }
}